src/test/groovy/io/codearte/jfairy/producer/person/PersonSpec.groovy

```diff
@@ -214,5 +231,118 @@ class PersonSpec extends Specification {
 			person.passportNumber()
 	}
 
+	def "withFirstName should create person with specific first name"() {
+		when:
+			def person = fairy.person(withFirstName("Specificfirstname"))
```

Comment:
Ok. Let's do it this way. I will merge it now. Please make another PR with def changes. Could you also include some instances where that has already been done.